pkg/server/server.go

```diff
@@ -52,15 +52,22 @@ func getScanningFilterFromConfigMap(filePath string) ([]string, error) {
 func readConfigScanningFilters() (skipList []string, err error) {
 	configDir := `/config`
 	configSkipListPath := filepath.Join(configDir, `skiplist`)
+	datreeSkipConfigDir := filepath.Join(configDir, `datreeSkipList`)
+	datreeSkipListPath := filepath.Join(datreeSkipConfigDir, `datreeSkipList`)
+	skipListPaths := []string{datreeSkipListPath, configSkipListPath}
+	skipLists := []string{}
+
+	for _, skipListPath := range skipListPaths {
+		if validateFileExistence(skipListPath) {
+			skipList, err = getScanningFilterFromConfigMap(skipListPath)
+			if err != nil {
+				return nil, err
+			}
+			skipLists = append(skipLists, skipList...)
```

Comment:
I think it's good to check we don't have the same expressions more than 1 time. The general list should contain each expression only once